tools/clang/test/DXC/Inputs/inc2.hlsli

```diff
@@ -0,0 +1,4 @@
+int f2(int g)
```

Comment:
I don't like having identical copies of the same files. Can the cmdtestfiles variants not be removed? Can the scripts there redirect to this location if nothing else?

Reply:
Can you confirm that these are all duplicates of what's in `cmdtestfiles`?

Reply:
The plan is to remove cmdtestfiles once https://github.com/microsoft/DirectXShaderCompiler/pull/4893 is done.
https://github.com/microsoft/DirectXShaderCompiler/pull/4893 depends on this PR.

Reply:
And yes. These are just copy of cmdtestfiles.